src/os/ui/state/stateimportui.js

```diff
@@ -140,7 +140,7 @@ os.ui.state.StateImportUI.prototype.showUI = function(file, config) {
     'label': 'Import State',
     'icon': 'fa fa-file-text',
     'x': 'center',
-    'y': 'center',
+    'y': '100',
```

Comment:
Why change / hard code this?

Reply:
Since the height is 'auto', I'm assuming the window position was firing too soon. Instead of hardcoding it, you can just emit the window ready event when the window will probably be ready to center. Its already being done in the stateimport.js constructor. You can always put `  $scope.$emit(os.ui.WindowEventType.READY);` in a timeout OR better yet in a resolve of a promise if its waiting for a network request or something

Reply:
Ya, that was the issue. Stateexport.js looks to have solved with a timeout so I added that solution to state import as well.